packages/did/src/DidLinks/AccountLinks.chain.ts

```diff
@@ -231,23 +232,6 @@ export async function getLinkRemovalByDidExtrinsic(
 
 /* ### HELPERS ### */
 
-function getMultiSignatureTypeFromKeypairType(
-  keypairType: KeypairType
-): SignatureType {
-  switch (keypairType) {
-    case 'ed25519':
-      return 'Ed25519'
-    case 'sr25519':
-      return 'Sr25519'
-    case 'ecdsa':
-      return 'Ecdsa'
-    default:
-      throw new SDKErrors.DidError(
-        `Unsupported signature algorithm "${keypairType}"`
-      )
-  }
-}
```

Comment:
This is not needed anymore

Reply:
As mentioned in my other comment, this will be extended with `Ethereum` and thus not be interoperable with what lives in the Did.utils

Reply:
My solution for that case would be adding another explicit mapping, which would have an upside of being very obvious.